### ISSUE TITLE
v7.15 Media picker 404s

### ISSUE THREAD

**Issue by richarth** [umbraco/Umbraco-CMS#5833]
After upgrading our sites to Umbraco 7.15 we are having issues with the media picker.  If you click on a media picker field to select an image you receive the following error:

```
Request error: The URL returned a 404 (not found): 
/umbraco/backoffice/UmbracoApi/Entity/GetById
```

The panel to select the file appears but there is either nothing in it or it gets stuck trying to load the files to show.

An example request URL:

`/umbraco/backoffice/UmbracoApi/Entity/GetById?id=umb%3A%2F%2Fmedia%2F3076da8b2ea94861ab8905b32c805a3b&type=media`

If we revert the site back to Umbraco 7.14 then the media picker works again.

**Comment by nul800sebastiaan** [comment]
We don't seem to have any problems with media pickers, so maybe there's some kind of configuration difference? I tried a few combinations of these but it all seems to work: 


![image](https://user-images.githubusercontent.com/304656/60965774-f108bb00-a316-11e9-8798-48963bdeaace.png)


It could be that you're having some upgrade problems, we did add the `Ignore user start nodes` option, might be what is going wrong for you.

Here's some tips on identifying upgrade problems: https://cultiv.nl/blog/how-to-diagnose-umbraco-upgrade-problems/

I'll close this for now, if you have more info on how to reproduce then feel free to comment!
